s3parq/session_helper.py

```diff
@@ -16,20 +17,34 @@ class SessionHelper:
     SH.configure_session_helper()
     '''
 
-    def __init__(self, region: str, cluster_id: str, host: str, port: str, db_name: str):
+    def __init__(self, region: str, cluster_id: str, host: str, port: str, db_name: str, ec2_user: str):
```

Comment:
i'm wondering if this will need to default to `None`?  this way we'll be able to instantiate this class without an `ec2_user`

Reply:
I think you're right that it might need to be None. I test this out